tinker/templates/profile.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'tinker_base.html' %}
+
+{% set title = 'Profile' %}
+
+{% block styles %}
+    <style type="text/css">
+        h2 {
```

Comment:
i would change all of these to be based on classes, instead of modifying any h2, th, td, li elements. It could mess up things like the nav and whatnot